features/sd-step.feature

```diff
@@ -15,30 +15,32 @@ Feature: Shared Steps
     Rules:
     - Package manager is available in every build.
     - Global SDK state is restored into the package manager installation/download location.
-    - Package manager can use the global SDK state to reduce installation/download time.
     - Method to update global SDK state.
     - Method to refer to installed packages via semver.
 
-    Background:
-        Given an existing pipeline with these images and packages with version:
-            | image          | package        | version        |
-            | golang:latest  | node           | ^4.0.0         |
-        And <image> image is used in the pipeline
-
-    Scenario: Use package via sd-step
+    Scenario Outline: Use package via sd-step
+        Given an existing pipeline with <image> image and <package> package:
+            | image   | package   |
```

Comment:
You don't really need this table anymore if you have <image> and <package> in the line above